roles/download/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ kube_version: v1.4.3
 etcd_version: v3.0.6
 #TODO(mattymo): Move calico versions to roles/network_plugins/calico/defaults
 # after migration to container download
-calico_version: v0.22.0
+calico_version: v1.0.0-beta
```

Comment:
#592 will be applied on top
calico/node 0.x can't work properly with SSL. It tries to send TLSv1.0 requests which are rejected. Only v1.0.0-beta works correctly. Therefore, the calico/node version is bumped